src/synchronization/intranode/mcs_lock.hpp

```diff
@@ -24,7 +24,7 @@ namespace locallock {
  * @warning The MCS need to be locked and unlocked by the same thread.
  */
 class mcs_lock {
-private:
+ private:
```

Comment:
The private/public(below) indentation is a tab in most other places, which is what I would normally expect.

Reply:
Let me undo this - as it does not belong in this PR